### ISSUE TITLE
Enhancement - Checking available commands before running the script

### ISSUE THREAD

**Issue by Moitaeel** [Witko/nvidia-xrun#164]
Hello mates,.

Recently when testing a fresh Fedora (36) install, I tried installing nvidia-xrun from copr on it to run openbox/awesome and whenever I tried running it I got a black screen and journal/Xorg-logs didn't show up anything wrong so it took me a while to understand what was happening until after killing openbox and seeing a message of a error on some line of nvidia-xinitrc which was an xrandr command, which then I realized xrandr wasn't installed.  

Not checking the dependencies was a noob error but it would be nice to have the nvidia-xrun script check if this command was available and send out an error message on absence before starting the X server and showing a black screen.
